docs/reference/system-requirements.rst

```diff
@@ -0,0 +1,41 @@
+.. _system-requirements:
+
+System requirements
+===================
+
+To run Snapcraft, a system requires the following minimum hardware and
+installed software. These requirements apply to local hosts as well as VMs and
+container hosts.
+
+
+Minimum hardware requirements
+-----------------------------
+
+- 64-bit processor
```

Comment:
This isn't technically true since we still release snaps for `armhf`, but I'm not sure it's worth quibbling about.

```suggestion
- Processor with a supported architecture (amd64, arm64, armhf, ppc64el, riscv64. s390x)
```

Reply:
a071107dd